tools/aqrtest.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+# project aquarium's testing battery
+# Copyright (C) 2021 SUSE, LLC.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+import sys
+import errno
+from pathlib import Path
+from libaqr.errors import BoxAlreadyExistsError, ImageNotFoundError, VagrantError
+from libaqr.images import Image
+from libaqr.runner import Runner
+from libaqr.suites import (
+    get_available_suites,
+    get_suite_tests
+)
+from libaqr.misc import find_builds_path
+from typing import Optional
+import click
+
+
+@click.group()
+def app() -> None:
+    pass
+
+
+@app.command("run")
+@click.argument("image", required=True, type=str)
+@click.option("-t", "--test", required=False, type=str)
+@click.option("-s", "--suite", required=False, type=str)
+@click.option("--suites-path", required=False, type=Path)
+def cmd_run(
+    image: str,
+    test: Optional[str],
+    suite: Optional[str],
+    suites_path: Optional[Path]
+) -> None:
+    """ Run a test or a suite against a given image """
+
+    suitesdir = suites_path if suites_path else Path("tests/suites")
+
+    if not suitesdir.exists():
+        click.secho("error: unable to find suites directory", fg="red")
+        sys.exit(errno.ENOENT)
```

Comment:
I think we've come across this before. Wouldn't it be better to use click's exit handling: https://click.palletsprojects.com/en/8.0.x/api/#click.Context.exit

This will also allow us to exit by raising appropriate exceptions etc.